src/bin.rs

```diff
@@ -8,12 +8,16 @@ extern crate clap;
 #[macro_use]
 extern crate log;
 
+extern crate base64;
+
 use clap::Clap;
 use std::fs::File;
 use std::fs::OpenOptions;
 use std::io::BufWriter;
 use std::sync::mpsc::{Receiver, Sender, TryRecvError};
 
+use structures::*;
+
```

Comment:
```suggestion
```
So that imports to the telemetry lib are grouped

Reply:
Done